explorer/src/Config.elm

```diff
@@ -74,6 +75,18 @@ update msg config =
         SearchComponentFocus value ->
             { config | searchHasFocus = value }
 
+        OnCross ->
```

Comment:
Would love some suggestion for a better name here! :D

Reply:
`OnClickClearSearchComponentInput` or something? 😄 Since this is the global msgs of the UIExplorer, it is nice to also indicate which component it affects